airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/StreamCopier.java

```diff
@@ -36,7 +36,7 @@
   /**
    * Writes a value to a staging file for the stream.
    */
-  void write(UUID id, AirbyteRecordMessage recordMessage) throws Exception;
+  void write(UUID id, AirbyteRecordMessage recordMessage, String fileName) throws Exception;
```

Comment:
The new `DatabricksStreamCopier` also implements this interface. Can you update that class as well?

Reply:
@tuliren, Thank you for paying attention to this. I updated `DatabricksStreamCopier`. Only I didn't update the version for this destination because I didn't find it in `airbyte-config`.

Reply:
Thank you for updating it. Currently the databricks connector is not public for legal reasons. Also this is not a functional change so not bumping the version is fine.

Reply:
@tuliren I didn't find her too. Can I add it to the airbyte-config? Or for the time being, put it off as it is?